amr-wind/CFDSim.H

```diff
@@ -117,6 +117,13 @@ public:
 
     bool has_mesh_mapping() const { return m_mesh_mapping; }
 
+    void during_overset_advance(const bool flag)
+    {
+        m_during_overset_advance = flag;
+    }
+
+    bool during_overset_advance() const { return m_during_overset_advance; }
```

Comment:
can you have them have different names? maybe this one should be `is_overset_advance` ? And the other should be `set_overset_advance`? Or maybe just plain `overset_advance`

Reply:
great idea

Reply:
I think I need them a little more verbose, though

Reply:
yeah good point.